### PR TITLE
Brand main to 9.0

### PR DESCRIPTION
**Pull request by carlossanlop**

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <VersionPrefix>3.11.0</VersionPrefix>
     <PreReleaseVersionLabel>beta1</PreReleaseVersionLabel>
-    <NetAnalyzersVersionPrefix>8.0.0</NetAnalyzersVersionPrefix>
+    <NetAnalyzersVersionPrefix>9.0.0</NetAnalyzersVersionPrefix>
     <NetAnalyzersPreReleaseVersionLabel>preview</NetAnalyzersPreReleaseVersionLabel>
```
Comment (carlossanlop) [inline eng/Versions.props:7]: There are two pre-release version labels @mmitche. One is "preview" as you suggested in the email, the other one is beta1, but I assume you'd like me to also set that to preview, correct?

**Comment by buyaa-n** [comment]
Looks like need to run `msbuild /t:pack`

**Comment by mmitche** [comment]
I'm not sure why they use two different sets of pre-release labels. IMO it doesn't make a lot of sense to me. @sharwell any insight here?

File: eng/Versions.props (hunk 1 above)
Comment (sharwell) [reply eng/Versions.props:7]: I wouldn't change the one that is set to `beta1`.

**Comment by carlossanlop** [comment]
@buyaa-n thanks for noticing the missing `msbuild /t:pack` output. I updated the PR.

@mmitche according to @sharwell in the comment, we should leave `beta1` untouched. So I think this PR might be ready.

Let's see what the CI says now.

**Review by buyaa-n — APPROVED** [review]
Thanks!

**Pull request merged**